tests/utils/test_parallel_progress.py

```diff
@@ -112,5 +112,6 @@ def assert_increasing(sequence: Sequence[float]) -> None:
     # longest task was programmed to take a known amount of time to run, so the
     # overall progress bar should have taken a max of ~ `max(task_lengths)` seconds.
     longtest_task_length = max(task_lengths)
-    # note: somewhat arbitrary offset here, was sometimes failing in the CI with just 1.
-    assert longtest_task_length <= total_time_seconds <= longtest_task_length + 1.5
+    # NOTE: This check was flaky. Adding some slack to the max time only temporarily
+    # addressed this.
+    assert longtest_task_length <= total_time_seconds  # <= longtest_task_length + 1.5
```

Comment:
If we want to test if we save time by doing parallel tasks maybe we could run some parallel `sleep 10` and make sure the total time is not larger than X times 10s?

Reply:
Fixed in b035f0610cde46be0a56004d4a39979f3c388391